webapp/public/index.html

```diff
@@ -24,7 +24,19 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
```

Comment:
We can load this dynamically; I'll show you how in the future, this is good for now (maybe make an issue as a reminder).